fast_llm/data/data/gpt/data.py

```diff
@@ -23,6 +27,20 @@
 logger = logging.getLogger(__name__)
 
 
+@dataclasses.dataclass
+class GPTBatch:
+    token_ids: torch.Tensor
+    loss_masking_spans: list[torch.Tensor] | None
+
+
+def gpt_data_collate_fn(batch: list[GPTSample], use_loss_masking_spans: bool) -> GPTBatch:
+    stacked_ids = np.stack([sample.token_ids for sample in batch])
+    stacked_spans = None
+    if use_loss_masking_spans:
+        stacked_spans = [torch.from_numpy(sample.loss_masking_spans) for sample in batch]
+    return GPTBatch(token_ids=torch.from_numpy(stacked_ids), loss_masking_spans=None)
```

Comment:
```suggestion
    return GPTBatch(token_ids=torch.from_numpy(stacked_ids), loss_masking_spans= stacked_spans)
```
sorry I gave you wrong code earlier :/